apps/minifront/src/components/ibc/layout.tsx

```diff
@@ -2,15 +2,46 @@ import { Card } from '@penumbra-zone/ui/components/ui/card';
 import { IbcInForm } from './ibc-in-form';
 import { IbcOutForm } from './ibc-out-form';
 
+import '@interchain-ui/react/styles';
+import { useStore } from '../../state';
+import { useChain } from '@cosmos-kit/react';
+import { ibcSelector } from '../../state/ibc';
+import { useLoaderData } from 'react-router-dom';
+import { useEffect } from 'react';
+import { Chain as PenumbraChain } from '@penumbra-zone/constants/src/chains';
+
 export const IbcLayout = () => {
+  useLoaderData();
+  const { penumbraChain } = useStore(ibcSelector);
+
+  console.log('## ibc-layout', penumbraChain?.chainName);
```

Comment:
useChain refuses to conditionally execute or execute with an empty chainName, but the way we've set up zustand forces a refresh with 'undefined' when changing the chainName state.